.travis.yml

```diff
@@ -8,4 +8,7 @@ branches:
   - master
 
 script:
-  util/repo
\ No newline at end of file
+  bundle exec rspec
```

Comment:
I thought it made sense to invoke rspec directly instead of creating a script with solely this line. My rule of thumb for stuff like this is to create a script if it is any more complicated than a single line.